textworld/utils.py

```diff
@@ -77,9 +79,16 @@ def chrome_driver(path=None):
         if path is not None:
             options.binary_location = path
 
-        driver = webdriver.Chrome(chrome_options=options)
+        SELENIUM_RETRIES = 10
+        SELENIUM_DELAY = 3  # seconds
+        for _ in range(SELENIUM_RETRIES):
+            try:
+                driver = webdriver.Chrome(chrome_options=options)
```

Comment:
Could just be `return webdriver.Chrome(...)`

Reply:
True :P